docs/user-guide/kubectl-conventions.md

```diff
@@ -35,7 +35,8 @@ In order for `kubectl run` to satisfy infrastructure as code:
 
 * Pod - use `run-pod/v1`.
 * Replication controller - use `run/v1`.
-* Deployment - use `deployment/v1beta1`.
+* Deployment (using `extensions/v1beta1` endpoint) - use `deployment/v1beta1`.
```

Comment:
One small nit. IMHO I think you should just drop the parentheses since the two would not be distinguishable otherwise, but I'll leave that up to you. LGTM otherwise.